ini.go

```diff
@@ -145,6 +145,12 @@ type LoadOptions struct {
 	// Relevant quote:  Values can also span multiple lines, as long as they are indented deeper
 	// than the first line of the value.
 	AllowPythonMultilineValues bool
+	// SpaceBeforeInlineComment indicates whether to allow comment symbols (\# and \;) inside value.
+	// Docs: https://docs.python.org/2/library/configparser.html
+	// Quote: Comments may appear on their own in an otherwise empty line, or may be entered in lines holding values or section names.
+	// In the latter case, they need to be preceded by a whitespace character to be recognized as a comment.
+	// (For backwards compatibility, only ; starts an inline comment, while # does not.)
```

Comment:
Is this line unnecessary? I see both ` #` and ` ;` added in the PR.

Reply:
Nope. Removed :+1: